smf/manifests/buckets-mdplacement.xml.in

```diff
@@ -10,8 +10,8 @@
     Copyright 2019 Joyent, Inc.
 -->
 
-<service_bundle type="manifest" name="-electric-boray">
-    <service name="manta/application/electric-boray" type="service" version="1">
+<service_bundle type="manifest" name="-buckets-mdplacement">
```

Comment:
That leading dash on `-buckets-mdplacement` doesn't look right. That was there before. I think it would be fine to remove it.

Reply:
Ok, I don't know why it was there either. I can remove it.